src/utils/resource-utils.ts

```diff
@@ -1,17 +1,17 @@
 import {
-  BcAlterationResource,
-  BenAlterationResource,
-  CccAlterationResource,
-  UlcAlterationResource
+  AlterationResourceBc,
+  AlterationResourceBen,
+  AlterationResourceCc,
+  AlterationResourceUlc
 } from '@/resources/Alteration'
 import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
 
 export class ResourceUtilities {
     private static entityTypeToResourceMap = {
-      [CorpTypeCd.BC_COMPANY]: BcAlterationResource,
-      [CorpTypeCd.BENEFIT_COMPANY]: BenAlterationResource,
-      [CorpTypeCd.BC_CCC]: CccAlterationResource,
-      [CorpTypeCd.BC_ULC_COMPANY]: UlcAlterationResource
+      [CorpTypeCd.BC_COMPANY]: AlterationResourceBc,
+      [CorpTypeCd.BENEFIT_COMPANY]: AlterationResourceBen,
+      [CorpTypeCd.BC_CCC]: AlterationResourceCc,
+      [CorpTypeCd.BC_ULC_COMPANY]: AlterationResourceUlc
```

Comment:
This map, and this class in general, is used for the _target_ entity type for an entity type alteration.

Hence, a business can (under the right circumstances) convert to a BC/BEN/CC/ULC. (They can convert _from_ a C/CBEN/CCC/CUL but not _to_ one of these.)